plugin/audit_log/buffer.cc

```diff
@@ -163,6 +163,7 @@ int audit_log_buffer_write(audit_log_buffer_t *log, const char *buf,
                       log_record_state_t::COMPLETE);
       audit_log_buffer_resume(log);
     }
+    audit_log_buffer_size_overflow++;
```

Comment:
Relax memory ordering from seq-cst to relaxed:
```suggestion
    audit_log_buffer_size_overflow.fetch_add(1, std::memory_order_relaxed)
```